### ISSUE TITLE
High CPU usage in VSCode when plotting

### ISSUE THREAD

**Issue by hafen** [randy3k/radian#286]
I have a strange issue that seems to have just started out of nowhere.

When I run radian inside vscode, everything is fine until I make a plot and a quartz device is opened. Then my CPU goes out of control and continues to do so until I quit R.

I have tried regular R inside vscode and there is no issue.

I also tried radian in the mac terminal and there is no issue.

Curious if anyone has experienced this or if there are any clues as to how I can troubleshoot. Radian and vscode have been an awesome combination.

I'm on MacOS 11.4, radian 5.11, R 4.1 (same issue with previous R version). I have disabled as many vscode extensions as possible. My setup is pretty vanilla.

<img width="795" alt="Screen Shot 2021-06-10 at 5 44 05 PM" src="https://user-images.githubusercontent.com/1275592/121614583-d748e480-ca13-11eb-9aa2-69413a83acec.png">

**Comment by renkun-ken** [comment]
Does it happen if use other graphics devices such as [httpgd](https://github.com/nx10/httpgd)?

**Comment by randy3k** [comment]
What version of radian it? There used to have a high cpu load [issue](https://github.com/randy3k/radian/issues/175) for an old version of radian but I believe that it was fixed.

**Comment by randy3k** [comment]
By the way, where could I find that Process Explorer? I couldn't find any information on google...

**Comment by hafen** [comment]
Thanks @randy3k. After quite a bit of digging I think I have found the culprit. It was a random VSCode setting:

```
"disable-hardware-acceleration": true
```

I changed that to `false` and that seems to have fixed the problem. I'm not sure when/how that got changed.

@renkun-ken thanks for pointing me to httpgd. That looks awesome! Kind of reminds me of a similar project I made a number of years ago, [rmote](https://github.com/cloudyr/rmote), but a million times better.
